fission-java-spring/pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   
   <groupId>io.fission</groupId>
-  <artifactId>fission-java-core</artifactId>
+  <artifactId>fission-java-spring</artifactId>
```

Comment:
In order to support backward compatibility - we should let `fission-java-core` be as it is